### DISCUSSION TITLE
Liquid Exception: incompatible character encodings: Windows-1252 and UTF-8 in /_layouts/default.html

### DISCUSSION THREAD

**Discussion by ybalkind** [mmistakes/minimal-mistakes#2581] [Q&A]
Hi 

my git repo is https://github.com/ybalkind/Yoni-Jeykll-Blog

When I copy my site and try to build it locally I get the error 
`Liquid Exception: incompatible character encodings: Windows-1252 and UTF-8 in /_layouts/default.html`

When Travis CI builds my site from github it gets a build error 

  `Error: could not read file /home/travis/build/ybalkind/Yoni-Jeykll-Blog/docs/vendor/bundle/ruby/2.4.0/gems/jekyll-3.8.7/lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb: Invalid date '<%= Time.now.strftime('%Y-%m-%d %H:%M:%S %z') %>': Document 'docs/vendor/bundle/ruby/2.4.0/gems/jekyll-3.8.7/lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb' does not have a valid date in the YAML front matter.
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    Invalid date '<%= Time.now.strftime('%Y-%m-%d %H:%M:%S %z') %>': Document 'docs/vendor/bundle/ruby/2.4.0/gems/jekyll-3.8.7/lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb' does not have a valid date in the YAML front matter.
The command "JEKYLL_ENV=production bundle exec jekyll build --destination site" exited with 1.`

I'm not sure if these two issues arise from the same cause or if its two separate issues?
I'm a newbie and am not a programmer

**Comment by mmistakes** [comment]
You'll have more luck expanding your search to [Jekyll's issue tracker](https://github.com/jekyll/jekyll/issues) or their [help forum](https://talk.jekyllrb.com/). This isn't a theme issue and has to do with Windows UTF encodings.

https://www.google.com/search?q=Liquid+Exception%3A+incompatible+character+encodings%3A+Windows-1252&rlz=1C5CHFA_enUS733US733&oq=Liquid+Exception%3A+incompatible+character+encodings%3A+Windows-1252&aqs=chrome..69i57j69i61l3.368j0j4&sourceid=chrome&ie=UTF-8

**Comment by ybalkind** [comment]
Okay thanks. 
Is the build error in Travis CI a separate issue? And if so should I open an issue for it here?

**Comment by mmistakes** [comment]
Think it’s related to the encoding. And if not it’s Jekyll/your environment related and not a theme issue. 

I’ve seen both many times before. Just search the exact error for suggested fixes. Windows can be tricky to get Ruby running properly. Same with some CIs.
